src/App.js

```diff
@@ -0,0 +1,46 @@
+import './App.css';
+import React from 'react';
+import {
+  BrowserRouter as Router,
+  Routes,
+  Route,
+  Link,
+} from 'react-router-dom';
+import Books from './redux/books/books';
+import Categories from './redux/categories/categories';
+
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+  }
+
+  render() {
+    return (
+      <Router>
+        <div>
+          <div>
+            <div>
+              Bookstore CMS
+            </div>
+            <nav>
+              <ul>
+                <li>
+                  <Link to="/">Books</Link>
+                </li>
+                <li>
+                  <Link to="/Categories">Categories</Link>
+                </li>
+              </ul>
+            </nav>
+          </div>
```

Comment:
- [ ] **[OPTIONAL]** To make our code look neater, I'd recommend that you extract this Navbar into a new component under `src/components` folder and render the component here instead.